charts/ratify/templates/assign.yaml

```diff
@@ -0,0 +1,89 @@
+{{- if semverCompare ">= 3.11.0" .Values.gatekeeper.version }}
+apiVersion: mutations.gatekeeper.sh/v1
+{{- else }}
+apiVersion: mutations.gatekeeper.sh/v1beta1
+{{- end }}
+kind: Assign
+metadata:
+  name: mutate-pod-image
+spec:
+  match:
+    scope: Namespaced
+    kinds:
+    - apiGroups: ["*"]
+      kinds: ["Pod"]
+    excludedNamespaces:
+      - "gatekeeper-system"
+      {{- if ne .Release.Namespace "gatekeeper-system"}}
+      - {{ .Release.Namespace }}
+      {{- end }}
+  applyTo:
+  - groups: [""]
+    kinds: ["Pod"]
+    versions: ["v1"]
```

Comment:
`applyTo` is an array, so can you just use a single `Assign` custom resource for all types?

Reply:
That would be ideal however workload resources have a different location which is why I created the 3 different providers.